### PR TITLE
+Implemented workaround to visualize remotely disconnected PVs

### PR DESCRIPTION
**Pull request by TheDuz**
+Added WebSocket PV subscription_access
  to keep updated PVs when only AccessRights change

### CODE REVIEW

**Comment by kasemir** [comment]
There's a merge conflict, as if you're missing the array_subscription code that was added ~2 months ago.

**Comment by kasemir** [comment]
OK, thanks.
Since you worry about access right changes, I assume you're enabling write access.
How are you handing security, since the access rights are based on the 'tomcat' user and not the actual end user?
Do you place tomcat behind a web proxy with authentication layer?

**Comment by TheDuz** [comment]
You've been too fast, my bad for creating a bad merge issue.

I'm using Access rights to prevent the write capability to some PVs when some process is undergoing

**Comment by kasemir** [comment]
Just note that access rights are based on the 'tomcat' user. PVWS doesn't know if user 'Fred' or user 'Jane' is in front of the web browser.

File: WebContent/js/pvws.js [hunk 1]
```diff
@@ -78,11 +78,19 @@ class PVWS
             if (value === undefined)
                 value = { pv: jm.pv, readonly: true };
             
-            // Update cached value with received changes
-            Object.assign(value, jm);
-            this.values[jm.pv] = value;
-            // console.log("Update for PV " + jm.pv + ": " + JSON.stringify(value));
-            this.message_handler(value);
+            //Only way found to show PV as disconnected
+            if(jm.value =="NaN")
```
Comment (kasemir) [inline WebContent/js/pvws.js:82]: Had to revert this.
A value may well be 'NaN'. Having one PV with that value must not result in closing the web socket.
The web socket is shared by many PVs, so with this change one NaN PV will disconnect all PVs, then reconnect, again get NaN, ... a never ending cycle.

**Comment by kasemir** [comment]
Unclear which problem you tried to fix with the "workaround to visualize remotely disconnected PVs".
They are shown fine.

Example display with 3 PVs from different sources:

Screen Shot 2020-10-22 at 9 49 00 AM

One IOC goes down, the PV from that IOC is shown with a border that indicates disconnect, and NaN for a value:

Screen Shot 2020-10-22 at 9 49 08 AM

While it's of course not ideal, PVs may occasionally disconnect because IOCs get restarted or as a result of network issues. The impacted PVs are indicated, other PVs continue to update. Closing the web socket means losing all PVs, which would be bad.

**Pull request merged**